recipe/meta.yaml

```diff
@@ -67,6 +67,8 @@ test:
     - blas =*=*{{ blas_impl }}
   commands:
     - f2py -h
+    - python -c "import sys; import numpy; sys.exit(0 if 'AVX512F? AVX512CD?' in numpy.lib.utils._opt_info() else 1)"  # [enforce_avx512 == "present" and blas_impl == "blis"]
+    - python -c "import sys; import numpy; sys.exit(0 if 'AVX512F* AVX512CD*' in numpy.lib.utils._opt_info() else 1)"  # [enforce_avx512 == "absent" and blas_impl == "blis"]
```

Comment:
You can directly access the `np.core.multiarray._multiarray_umath.__cpu_features__['AVX512F']`, and we should think of a nicer way to expose this from NumPy.

Reply:
Yeah, you're right, thanks (I now remember a similar discussion from https://github.com/numpy/numpy/issues/18058). But now that the test suite passes with blis both with/without avx512, I'll be able to drop those conditions again anyway.